frameworks/FEDOT/exec_ts.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+from pathlib import Path
+import numpy as np
+
+from fedot.api.main import Fedot
+from fedot.core.repository.tasks import Task, TaskTypesEnum, TsForecastingParams
+from fedot.core.data.data import InputData
+from fedot.core.repository.dataset_types import DataTypesEnum
+
+from frameworks.shared.callee import call_run, result, output_subdir
+from frameworks.shared.utils import Timer, load_timeseries_dataset
+
+log = logging.getLogger(__name__)
+
+
+def run(dataset, config):
+    log.info("\n**** FEDOT ****\n")
+
+    scoring_metric = get_fedot_metrics(config)
+
+    training_params = {"preset": "best_quality", "n_jobs": config.cores}
+    training_params.update({k: v for k, v in config.framework_params.items() if not k.startswith('_')})
+    n_jobs = training_params["n_jobs"]
+
+    log.info('Running FEDOT with a maximum time of %ss on %s cores, optimizing %s.',
+             config.max_runtime_seconds, n_jobs, scoring_metric)
+    runtime_min = config.max_runtime_seconds / 60
+
+    task = Task(
+        TaskTypesEnum.ts_forecasting,
+        TsForecastingParams(forecast_length=dataset.forecast_horizon_in_steps)
+    )
+
+    train_df, test_df = load_timeseries_dataset(dataset)
+    id_column = dataset.id_column
+
+    log.info('Predicting on the test set.')
+    training_duration, predict_duration = 0, 0
+    models_count = 0
+    truth_only = test_df[dataset.target].values
+    predictions = []
+
+    for label in train_df[id_column].unique():
+        train_sub_df = train_df[train_df[id_column] == label].drop(columns=[id_column], axis=1)
+        train_series = np.array(train_sub_df[dataset.target])
+        train_input = InputData(
+            idx=train_sub_df.index.to_numpy(),
```

Comment:
This code uses the range index `[0, 1, 2, ..., n-1]` instead of the timestamps of the time series. Should this be `train_sub_df[timestamp_column]` instead, or does FEDOT ignore the timestamps?

Reply:
FEDOT ignores timestamps, so that idx should be fine